string agent
string facility
string action
string destination
string further_action
string item
string job_id
string part_of
uint32 amount
uint32 steps
float32 task_depth # describing the task dependecy depth level
string task_id
string parent_task_id
uint32 rating
string[] required_tools
string best_workshop
string sim_id
